UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentDataService.cs

```diff
@@ -85,11 +85,21 @@ public async Task<string> CreateExchangeSet(SalesCatalogueServiceResponseQueueMe
 
             if (aioConfiguration.IsAioEnabled)
             {
+                SalesCatalogueDataResponse catalogueDataResponse = (SalesCatalogueDataResponse)salesCatalogueEssDataResponse.Clone();
```

Comment:
```suggestion
                SalesCatalogueDataResponse salesCatalogueEssDataResponseForAio = (SalesCatalogueDataResponse)salesCatalogueEssDataResponse.Clone();
```
Can we change name as suggested to make variable meaningful?